tests/apps/ideas/models.py

```diff
@@ -21,9 +21,9 @@ class Idea(Item):
     name = models.CharField(max_length=120,
                             default='Can i haz cheezburger, pls?')
     description = RichTextField(verbose_name='Description', blank=True)
-    moderator_feedback = models.CharField(max_length=256, blank=True)
-    moderator_statement = models.OneToOneField(
-        ModeratorStatement,
+    moderator_status = models.CharField(max_length=256, blank=True)
+    moderator_feedback_text = models.OneToOneField(
```

Comment:
To me `moderator_feedback` makes more sense here? And then the text is `moderator_feedback.feedback_text` .. why did you name it `moderator_feedback_text`?